packages/firebase/jest.config.js

```diff
@@ -1 +1,6 @@
-module.exports = require("../../jest.config")
+module.exports = {
+  preset: "ts-jest",
+  testEnvironment: "node",
+  modulePathIgnorePatterns: ["dist/", "node_modules/"],
+  resolver: "jest-node-exports-resolver",
+}
```

Comment:
Please continue to require the root `jest.config.js`, you can extend it wiht your resolver if needed in this case like this:

```
const jestConfig = require('../../jest.config')

module.exports = {
  ...jestConfig,
  resolver: "jest-node-exports-resolver",
}
```